install/updateVersion.sh

```diff
@@ -125,6 +125,9 @@ function merge_files() {
   cp $ISTIO $ISTIO_AUTH
   sed -i=.bak "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" $ISTIO_AUTH
 
+  # Deploy istio-ca always, required to enable MTLS for mixer and pilot
```

Comment:
Move this before copying ISTIO into ISTIO_AUTH, their content is identical except for the MUTUAL_TLS.

But copy to ISTIO_CLUSTER_WIDE before the CA, because it gets a different CA.